server/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleServer/server/JungleDB.java

```diff
@@ -45,4 +41,9 @@ public Connection getConnection() throws SQLException {
             throw new SQLException("Data source is not initialized.");
         }
     }
+
+    private void initDataSource(Properties properties) {
+        HikariConfig config = new HikariConfig(properties);
+        dataSource = new HikariDataSource(config);
+    }
```

Comment:
Updated to use the HakariCP sql connection pooler. It's a lot simpler to use, and it makes databases a lot more pluggable. The hope was I could leverage this to setup an entirely in-memory DB for running tests. Still a WIP.